integrationTests/testProcessorNode.go

```diff
@@ -1536,6 +1537,7 @@ func (tpn *TestProcessorNode) initInnerProcessors(gasMap map[string]map[string]u
 		TestMarshalizer,
 		TestHasher,
 		tpn.ShardCoordinator,
+		&commonMocks.TxExecutionOrderHandlerStub{},
```

Comment:
suggestion to use a real tx execution order handler in all TestProcessorNodes variants (create an exported field on the TestProcessorNode and use that field just about everywhere)

Reply:
done